src/services/doppler-api-client.double.ts

```diff
@@ -49,34 +49,29 @@ const subscriber = {
   score: 2,
 };
 
-const campaignDeliveryCollection = {
-  items: [
-    {
-      campaignId: 1,
-      campaignName: 'Campaña estacional de primavera',
-      campaignSubject: '¿Como sacarle provecho a la primavera?',
-      deliveryStatus: 'opened',
-      clicksCount: 2,
-    },
-    {
-      campaignId: 2,
-      campaignName: 'Campaña calendario estacional 2019',
-      campaignSubject: 'El calendario estacional 2019 ya está aquí',
-      deliveryStatus: 'notOpened',
-      clicksCount: 23,
-    },
-    {
-      campaignId: 3,
-      campaignName: 'Emms 2019 preveento 1',
-      campaignSubject: 'Ya comienza el dia 2. Accede a las conferencias',
-      deliveryStatus: 'softBounced',
-      clicksCount: 100,
-    },
-  ],
-  currentPage: 0,
-  itemsCount: 3,
-  pagesCount: 1,
-};
+const campaignDeliveryItems = [
```

Comment:
I think here perhaps you could add more data, for us to see how this goes, visually. What do you think?

Reply:
yes i added now